app/lib/frontend/handlers/account.dart

```diff
@@ -45,16 +46,16 @@ Future<shelf.Response> updateSessionHandler(
   if (userSessionData != null &&
       userSessionData.userId == user.userId &&
       userSessionData.email == user.email &&
-      userSessionData.imageUrl == clientSessionData.imageUrl) {
+      userSessionData.imageUrl == body.imageUrl) {
```

Comment:
For the record, if we want to get the `imageUrl` from oauth2, we can do that using:
https://pub.dev/documentation/googleapis/latest/googleapis.oauth2.v2/UserinfoResourceApi/get.html

Example (not tested):
```dart
import 'package:googleapis/oauth2/v2.dart' as googleapis;
import 'package:googleapis_auth/auth_io.dart' as auth;

final client = http.Client();
final authClient = auth.authenticatedClient(client, AccessCredentials(accessToken: '<accessToken>', '', []));

final oauth2 = googleapis.Oauth2Api(authClient);
final userinfo await = oauth2.userinfo.get();
await client.close();

print(userinfo.id); // oauth2 ID
print(userinfo.name);
print(userinfo.email);
print(userinfo.picture);
```

We don't have to do this now, feel free to file this as a follow up item.

Reply:
I'll do this as a follow-up.